src/test_l1_dsCompositeIn.c

```diff
@@ -286,21 +287,18 @@ void test_l1_dsCompositeIn_positive_dsCompositeInGetNumberOfInputs(void)
 {
     gTestID = 5;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
```

Comment:
Please correct the Test Procedure description at line number 280. Replace Step 2, Step 3, and Step 4 with a proper description that includes validating the number of composite ports from the configuration YAML file. Also check whether  compilation is fine

Reply:
done